packages/wrangler/src/pages.tsx

```diff
@@ -284,7 +289,7 @@ function generateRulesMatcher<T>(
     T
   ][];
 
-  return ({ request }: { request: Request }) => {
+  return ({ request }: { request: MiniflareRequest }) => {
```

Comment:
Out of curiosity (OOC) - why the need to change these types?

Reply:
They're the same types as before, just renamed. I'm not sure as to why, @GregBrimble ?

Reply:
Oh right